tokenserver/assignment/sqlnode/sql.py

```diff
@@ -701,7 +693,7 @@ def get_best_node(self, service, email):
         # This is a little racy with concurrent assignments, but no big deal.
         where = [nodes.c.service == service, nodes.c.node == node]
         where = and_(*where)
-        fields = {'available': nodes.c.available - 1,
+        fields = {'available': self._sqlfunc_max(nodes.c.available - 1, 0),
```

Comment:
This change has the advantage of correctly maintaining `current_load` for the spanner node, but the risk of underflowing `available` because we ignore its `capacity`. Don't allow it to underflow.

Reply:
well, since this is updating the node info, and spanner is a special case, treating spanner as a special case and not updating the nodes when we send an account.